increasing_height_order.py

```diff
@@ -0,0 +1,61 @@
+from nptyping import NDArray
+from numpy import argsort, array
+
+
+class Student:
+    """Class student.
+    :param identifier: Student's id.
+    :param height: Student's height.
+
+    """
+
+    def __init__(self, identifier:str, height: float):
+        """Constructor
+        """
+        self.identifier = identifier
+        self.height = height
+
+    def __lt__(self, other: 'Student'):
+        """Receive another student and return true if its height > current student.
+
+        :param other: Other Student object.
+        :return: True if left student's height < other student's height.
+        """
+        return self.height < other.height
```

Comment:
When accessing a private attribute of a class you should use getters and setters.

Reply:
Hi Tal,

I disagree with you because this line is inside a method of the class.

I'll be happy if you look at it again.

Thanks in advance.